testingutils/src/main/java/com/github/awsjavakit/testingutils/ApiGatewayRequestBuilder.java

```diff
@@ -48,6 +49,16 @@ public <I> ApiGatewayRequestBuilder withBody(I body) {
     }
   }
 
+  /**
+   * Add the query path.
+   * @param path the path.
+   * @return the builder.
+   */
+  public ApiGatewayRequestBuilder withPath(UnixPath path) {
```

Comment:
This class has no tests yet. Probably just constructing the event would be the test